### DISCUSSION TITLE
I have a computer that runs version 69 normally, and the version above 69 has the phenomenon that the embedded webpage freezes and cannot be operated. But other computers are operating normally.

### DISCUSSION THREAD

**Discussion by wutangyuan** [cefsharp/CefSharp#3491] [General]
- **What version of the product are you using?**
    v89.0.170

- **What architecture x86 or x64?**
    x64
	
- **What version of .Net?**
    .Net 4.7.2
    
- **On what operating system?**
   Win10

- **Are you using `WinForms`, `WPF` or `OffScreen`?**
    WPF
    
Detail：
 I have a computer that runs version 69 normally, and the version above 69 has the phenomenon that the embedded webpage freezes and the webpage cannot be operated. But other computers are operating normally. I tried to install various versions of vc++ libraries and NetCore runtime libraries on that computer, but none of them worked

**Comment by amaitland** [comment]
Have you checked the log file? https://github.com/cefsharp/CefSharp/wiki/Trouble-Shooting#log-file

What do you mean by freezers? Does it load a web page then stop responding? Please describe what is happening in more detail. 

What version of VC++ do you have installed? 

What's different about this computer to the working ones? Hardware? Software?
